types/node-pixel/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "private": true,
+    "dependencies": {
+        "johnny-five": "^2.1.0"
```

Comment:
You probably just need `@types/johnny-five` in here and nothing else. Check out one of the other packages in the repo for an example.